internal/services/service.go

```diff
@@ -0,0 +1,19 @@
+package services
+
+import (
+	"context"
+
+	"example.com/tfgrid-kyc-service/internal/responses"
+)
+
+type TokenService interface {
```

Comment:
Are these defined now for the sake of testing later? Not sure if they add anything yet.

Reply:
yes